requirements.txt

```diff
@@ -1 +1,3 @@
 molo.core==2.4.0
+
```

Comment:
Please remove the empty line, I'm not sure if we filter out these when we read the file in the `setup.py` file.